mayastor/tests/common/error_bdev.rs

```diff
@@ -22,7 +18,9 @@ pub fn create_error_bdev(error_device: &str, backing_device: &str) {
     let err_bdev_name_str = std::ffi::CString::new(error_device.to_string())
         .expect("Failed to create name string");
     unsafe {
-        retval = spdk_vbdev_error_create(err_bdev_name_str.as_ptr()); // create the error bdev around it
+        retval = vbdev_error_create(err_bdev_name_str.as_ptr()); // create the
```

Comment:
nit: I'd suggest adding the comment as one line above

Reply:
I had that but rust-fmt disagrees.